examples/multi-project/README.md

```diff
@@ -112,14 +115,13 @@ jobs:
           apiKey: $(infracostApiKey)
 
       - bash: |
-          infracost output --path="infracost_jsons/*.json" --format=json --out-file=/tmp/infracost_combined.json
-        displayName: Combine the results
-
-      - task: InfracostComment@0
-        displayName: Post the comment
-        inputs:
-          githubToken: $(githubToken)
-          path: /tmp/infracost_combined.json
-          behavior: update # Create a single comment and update it. See https://github.com/infracost/infracost-azure-devops#infracostcomment for other options
+          # Create a single comment and update it. See https://www.infracost.io/docs/features/cli_commands/#comment-on-pull-requests for other options
+          infracost comment github \
+          --path infracost_jsons/*.json \
```

Comment:
```suggestion
          --path "infracost_jsons/*.json" \
```

Sorry, put the comment in the wrong place. This is the CI fix, need to regenerate the examples.

Reply:
nice! fixed in https://github.com/infracost/infracost-azure-devops/pull/45/commits/4fa1b8f25a406aab41ae47519cc8f587351957d1